src/core/screen.js

```diff
@@ -131,7 +134,11 @@ export class Screen extends Phaser.Scene {
         this._layout.makeAccessible();
         this.sys.accessibleButtons.forEach(button => a11y.addButton(button));
         a11y.reset();
-        this.setStatsScreen(this.scene.key);
+        if (this.scene.key.includes("shop")) {
+            this.setStatsScreen(this.transientData.shopTitle + "menu");
+        } else {
+            this.setStatsScreen(this.scene.key);
+        }
```

Comment:
`this.setStatsScreen(this.scene.key.includes("shop") ? this.transientData.shopTitle + "menu" : this.scene.key`
probably can do in 1 line

Reply:
`if` seems fine here to me, I don't think it reads any better as a one liner

Reply:
I do tend to strive for less code / lines though as it helps with readability of the whole file.
I've possibly gone a bit too far down the"if staements are a code smell" lately though :)